src/content/docs/en/pages/deploy-journey/install-orch-agent/install-orch-agent.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Install Edge Orchestrator Agent
+description: To begin the Edge Node installation process, you must download the Edge Orchestrator installation binary of your choice.
+meta_tags: deploy, orchestrator
+namespace: docs_deploy_install_agent
+menu_namespace: deployMenu
+permalink: /documentation/products/guides/deploy/install-orchestrator-agent/
+---
+
+import Button from '~/components/Button.astro'
+
+:::note
+Before installing the agent, it's necessary to generate a credential.
+:::
+
+<Button href="/en/documentation/products/deploy/credentials/" text="go to credentials reference documentation" variant="secondary" target="_blank"></Button>
+
+
+To begin the **Edge Node** installation process, you must download the **Edge Orchestrator** installation binary of your choice.
+
+Download the Azion Edge Node option that matches your platform of use:
+
+| Operating System | Architecture | File                                                         |
+| :--------------- | :----------: | ------------------------------------------------------------ |
+| FreeBSD          |    x86_64    | [edge-orchestrator](https://downloads.azion.com/freebsd/x86_64/edge-orchestrator) |
+| FreeBSD          |    ARM64     | [edge-orchestrator](https://downloads.azion.com/freebsd/arm64/edge-orchestrator) |
+| Linux            |    x86_32    | [edge-orchestrator](https://downloads.azion.com/linux/x86_32/edge-orchestrator) |
+| Linux            |    x86_64    | [edge-orchestrator](https://downloads.azion.com/linux/x86_64/edge-orchestrator) |
+| Linux            |    ARM32     | [edge-orchestrator](https://downloads.azion.com/linux/arm32/edge-orchestrator) |
+| Linux            |    ARM64     | [edge-orchestrator](https://downloads.azion.com/linux/arm64/edge-orchestrator) |
+| MacOS            |    x86_64    | [edge-orchestrator](https://downloads.azion.com/darwin/x86_64/edge-orchestrator) |
+
+Alternatively, you can download it using the Command line. In the example, the download link refers to the Linux/x86_64 agent. If you want to download another version, just change the URL for the required operating system or architecture:
+
+`curl -O http://downloads.azion.com/linux/x86_64/edge-orchestrator`
+
+> **Note**: The commands executed in the edge-orchestrator agent must be executed with root user privileges. It might be necessary to run the commands with `sudo`.
```

Comment:
```suggestion
:::note
The commands executed in the edge-orchestrator agent must be executed with admin user privileges. It might be necessary to run the commands with `sudo`.
:::
```